pwa/src/components/endpoints/endpointForm.tsx

```diff
@@ -121,156 +118,147 @@ export const EndpointForm: React.FC<EndpointFormProps> = ({id}) => {
       .then((data) => {
         setLoadingOverlay(false);
         setEndpoint(data)
-        method === 'POST' && navigate("/endpoints")
+        method === 'POST' && navigate("/endpoints") && setAlert({message: 'Saved endpoint', type: 'success'})
```

Comment:
Why are we chaining everything with `&&` here? I think it might be better to keep them on separate lines.